packages/connectivity/src/scp-cf/connectivity-service.ts

```diff
@@ -111,7 +111,7 @@ async function proxyHeaders(
 }
 
 async function proxyAuthorizationHeader(
-  jwt?
+  jwt?: string
```

Comment:
Good change, but why here?

Reply:
I don't remember 😅 I think I had that file still open when I started working on this ticket